docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md

```diff
@@ -203,10 +203,17 @@ See [Token description tables](#token-description-tables) for more information.
 
 **Promotional product discount**
 
-This discount type lets you discount a specific product at a fixed quantity whenever the discount conditions are met. A common use case is for a "buy x, get y" discount where a customer gets an item for free when they buy a certain product or spend a certain amount. The product you are promoting goes in the **Abstract product SKU** field. The **Quantity** field represents the limit on the number of items eligible for the discount. When a customer fulfills the discount conditions, the promotional product is automatically merchandised below the items in a customer's cart. The customer can either add the product to the cart from this widget or any other place in the shop, and the discount will apply. It is important to note that the promotional product discount only applies if the product is added to the cart after meeting the discount's conditions. If the product is already in the cart before the discount conditions are met, the customer needs to remove and re-add it.
-![Application type](https://spryker.s3.eu-central-1.amazonaws.com/docs/User+Guides/Back+Office+User+Guides/Discount/Discount+Calculation:+Reference+Information/Application+type.png)
+This discount type lets you discount specific products at a fixed quantity whenever the discount conditions are met. A common use case is for a "buy x, get y" discount, where a customer gets an item for free when they buy a certain product or spend a certain amount. When you create a promotional discount in the Back Office, in the **ABSTRACT PRODUCT SKU(S)** field, you enter comma-separated abstract product SKUs—products you are promoting. The **MAXIMUM QUANTITY** field represents the limit on the number of items eligible for the discount. When a customer fulfills the discount conditions, the promotional products are automatically merchandised below the items in a customer's cart. For example, there are 10 SKUs in **ABSTRACT PRODUCT SKU(S)**, and **MAXIMUM QUANTITY** is set to `1`. If the customer meets the discount conditions, they are eligible for only 1 unit of any of the 10 promotional product SKUs (NOT 1 of each). The customer can add one of these products to the cart from this widget or any other place in the shop, and the discount will apply.
 
-You can either give away the promotional product completely for free or provide a discount for this product by specifying the percentage value or a fixed amount to be discounted from the promotional product's price (when giving the product for free, the percentage value should be 100%, while the fixed-price value should be equal to the product's price).
+{% info_block infoBox "Note" %}
+
+The promotional product discount only applies if the product is added to the cart after meeting the discount's conditions. If the product is already in the cart before the discount conditions are met, the customer needs to remove and re-add it.
```

Comment:
```suggestion
The promotional product discount only applies if the product is added to the cart from the Promotional Products cart widget. If the product is already in the cart before the discount conditions are met, the customer needs to remove and re-add it from the widget.
```

Reply:
This only applies to the Yves implementation of promotional products. With GLUE API this behavior is more customizable. Would we include that documentation here as well?